app/helpers/application_helper.rb

```diff
@@ -18,4 +18,31 @@ def last_edited_by(resource)
 
     "Last edited by #{resource.last_edited_by.username}"
   end
+
+  # Map of status to CSS class names
+  STATUS_CLASS_MAPPING = {
```

Comment:
You can use the class_names helper for these sorts of things https://www.bigbinary.com/blog/rails-6-1-introduces-class_names-helper :)